genie/validate.py

```diff
@@ -108,7 +111,11 @@ def validate_single_file(self, **kwargs):
                 syn=self._synapse_client,
                 center=self.center,
                 genie_config=self.genie_config,
+                ancillary_files=self.ancillary_files,
             )
+            print("------Printing ancillary files------")
+            print(validator.ancillary_files)
```

Comment:
Nit: I think the PR to actually use the ancillary files actually covers this print statement